contrib/process/runnable.go

```
@@ -33,5 +33,5 @@ func (p *process) Run(ctx context.Context) error {
 	defer cancel()
 
 	<-ctx.Done()
-	return ctx.Err()
+	return nil
```

Don't we want to return an error saying that the process was cancelled due to a sigterm?

---

Oh, nevermind, this is strictly the process runnable.